articles/portal/ptl-how-use-my-calls.md

```diff
@@ -251,6 +251,8 @@ You can enable new users on your account with access to My Calls.
 
 5. To provide the user with access to My Calls, you must raise a service request using the **Setting up a user in My Calls** offering in the Service Catalog.
 
+    ![New User Request Offering](images/ptl-mycalls-new-user-request-offering.png)
```

Comment:
Text in the image is "New user setup in My Calls", so could you change the step above to reflect that?